packages/passport/sdk-sample-app/src/components/zkevm/EthSignTypedDataV4Examples/SeaportCreateListing.tsx

```diff
@@ -55,22 +58,46 @@ function SeaportCreateListing({ disabled, handleExampleSubmitted }: RequestExamp
   useEffect(() => {
     setSignMessageError('');
     setTransaction(undefined);
-  }, [NFTContractAddress, tokenContractAddress, buyAmount, buyType, tokenId]);
+  }, [NFTContractAddress, tokenContractAddress, buyAmount, buyType, tokenId, sellTokenUnits, sellTokenType]);
+
+  const resetForm = () => {
+    setSellTokenType('ERC721');
+    setNFTContractAddress('');
+    setTokenContractAddress('');
+    setBuyAmount('');
+    setTokenId('');
+    setBuyType('NATIVE');
+    setSellTokenUnits('1');
+    setShowSellTokenUnitsField(false);
+  };
 
   const validate = useCallback(async () => {
     const buy = buyType === 'NATIVE'
       ? { amount: buyAmount, type: buyType }
       : { amount: buyAmount, type: buyType, contractAddress: tokenContractAddress };
+    const sell = sellTokenType === 'ERC721'
+      ? {
+        contractAddress: NFTContractAddress,
+        tokenId,
+        type: 'ERC721',
+      } as ERC721Item : {
+        contractAddress: NFTContractAddress,
+        tokenId,
+        type: 'ERC1155',
+        amount: sellTokenUnits,
+      } as ERC1155Item;
+
     try {
       setIsBuildingTransaction(true);
+
+      if (sellTokenType === 'ERC1155' && sellTokenUnits === '0') {
+        throw new Error('Units for sale must be greater than 0');
```

Comment:
I think local throwing of errors is fine but this did show up as a warning in the IDE static code analysis. If there's strong opinion against this then can consider changing.

Reply:
If primary use case of the sample app is local testing I'd think it's fine